core/arch/arm/plat-rzg/conf.mk

```diff
@@ -0,0 +1,47 @@
+PLATFORM_FLAVOR ?= hihope_rzg2m
+
+include core/arch/arm/cpu/cortex-armv8-0.mk
+
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+$(call force,CFG_SCIF,y)
+$(call force,CFG_CORE_LARGE_PHYS_ADDR,y)
+$(call force,CFG_CORE_ARM64_PA_BITS,36)
+
+# Disable core ASLR for two reasons:
+# 1. There is no source for ALSR seed, as RZG platform
+#    does not provide DTB to OP-TEE. Also, there is no
+#    publicly available documentation on integrated
+#    hardware RNG, so we can't use it either.
+# 2. OP-TEE crashes during boot with enabled CFG_CORE_ASLR.
+$(call force,CFG_CORE_ASLR,n)
+
+ifeq ($(PLATFORM_FLAVOR),ek874)
+$(call force,CFG_TEE_CORE_NB_CORE,2)
+endif
+ifeq ($(PLATFORM_FLAVOR),hihope_rzg2m)
+$(call force,CFG_TEE_CORE_NB_CORE,6)
+
```

Comment:
This blank line looks weird (splitting the `ifeq/endif` blocks in a non-logical way), would you mind removing it?

Reply:
Agreed will drop that.